File: discover_more_python/github_most_popular.py
from urllib2 import Request, urlopen, URLError

request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token 7e573dd60371dc91c35929139cdca698e477e85c'
}

request = Request('https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc')

try:
	response = urlopen(request)
	read_file = response.read()
	print read_file
except URLError, e:
    print 'No file. Got an error code:', e

File: discover_ruby/github_most_popular.rb
#!/usr/bin/ruby
require 'HTTPClient'

extheaders = {
  'User-Agent' => 'Holberton_school',
  'Authorization' => '5b0fbcd1114ea6d6c375c414f1d48d87ef0b71bb'
}

http = HTTPClient.new
puts http.get_content ('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc')

